app.js

```diff
@@ -25,8 +30,14 @@ if (question1 === 'N') {
 } else {
   alert('Seriously? Even after I said no bullshit? Come on man! Alright, you lose. 0 for 0 so far, happy now?');
 }
+};
+
+question1Function();
```

Comment:
I would wait and call these functions at the end of the file, makes it easier to find all of them if you need to change the call order or stop calling one of the functions.